tailwind.config.js

```diff
@@ -32,6 +32,13 @@ module.exports = {
           DEFAULT: "hsl(var(--muted))",
           foreground: "hsl(var(--muted-foreground))",
         },
+        theme: {
+          extend: {
+            colors: {
+              lavender: "#E6E6FA", // Added
+            },
+          },
+        },
```

Comment:
Using themes is a great idea! I encourage you to play around with them, but for organization purposes let's wait until the design is finalized to add them to the main codebase